z3c/dependencychecker/main.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from z3c.dependencychecker.dependencychecker import logger
+from z3c.dependencychecker.dependencychecker import parse_command_line
+from z3c.dependencychecker.dependencychecker import set_log_level
+
+import os
+import pkg_resources
+import sys
+
+
+def main():
+    options, args = parse_command_line()
+    set_log_level(options.verbose)
+    path = get_path(args)
+    get_package_dependencies(path)
+
+
+def get_path(args):
+    """Get path from given arguments or use """
+    path = os.getcwd()
+    if len(args) > 0:
+        path = os.path.abspath(args[0])
+        if not os.path.isdir(path):
+            logger.info('Given path does not exist: {0}'.format(args[0]))
+            sys.exit(1)
+    logger.info('path used: %s', path)
+    return path
+
+
+def get_package_dependencies(path):
+    if not has_setup_py_file(path):
+        logger.info('No setup.py file found! on {0}'.format(path))
+        sys.exit(1)
+
+    new_path, filename = get_egg_info_folder(path)
+    if not new_path:
+        logger.info('No egg-info folder found! on {0}'.format(new_path))
+        sys.exit(1)
+    path = new_path
+    package_name = filename[:-len('.egg-info')]
+
+    working_set = pkg_resources.WorkingSet()
+    working_set.add_entry(path)
+    new_requirement = pkg_resources.Requirement.parse(package_name)
```

Comment:
I notice I find it a bit hard to understand what "new" is. `new_requirement`, `new_path`.
Names like `our_package_name`, `our_own_requirement`, `our_egg_info_path` (just to give an idea) make it clearer to follow what happens.

Reply:
Yeah this last chunk of code was copy & pasted directly from an interactive terminal 🙂 I will change them to proper names, function names as well and some docstrings.